src/amuse/units/optparse.py

```diff
@@ -1,4 +1,8 @@
-
+print(
```

Comment:
Should this use `warnings.warn()` instead of `print()`?

Reply:
Yep. But couldn't get that to work from the train. Feel free to fix.